mowgli_etl/pipeline/wdc/Data_Exploration/offers_corpus_english_v2_clean.py

```diff
@@ -0,0 +1,47 @@
+from os import listdir
```

Comment:
Use PEP8 everywhere, please. Including file and directory names -- data_exploration.